.github/workflows/ci-dgraph-tests.yml

```diff
@@ -81,10 +81,11 @@ jobs:
           export GOPATH=~/go
           # move the binary
           cp dgraph/dgraph ~/go/bin/dgraph
-          # run the tests
+          # Run oss unit tests
+          go test -timeout=60m -failfast -tags=oss ./...
```

Comment:
By adding this line you are running these tests twice (they should already get picked up by `./t`)

Reply:
If the goal is to make sure that OSS builds don't fail in the future, we should add another (parallel) workflow, so this job doesn't get even longer than it already is.

Reply:
These are the same unit tests but now running for OSS build. And these are only unit tests, they only take a few seconds to run. I want to make sure any PR we have in future does not break at least building the OSS release. I agree that we should have a separate workflow, but that may be a bigger effort where we should be able to run t binary tests for OSS build. I.can file a JIRA issue for it.

Reply:
On my local machine it takes
```
real	0m36.089s
```
to run these units tests, so a nontrivial amount of time IMO.  We are paying to run these machines, so ideally we should aim to reduce the time it takes to run these tests.  It is not a huge effort to create a new workflow to run these tests (on github hosted runners).  Also adding this line is confusing because as I mentioned we are already running these tests when we call `./t`.  I think someone seeing this would be confused.  

Otherwise this PR looks fine, we can merge it and then I can add the workflow.

Reply:
@mangalaman93 See https://github.com/dgraph-io/dgraph/pull/8834.  Once this is merged I can rebase and merge the other PR.

Reply:
cool, that makes sense. removed this change now. Thanks for the review and the PR, this PR is good to go now.